tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Logging.cs

```diff
@@ -52,7 +52,6 @@ internal static partial class ConfigurationKeys
 
         /// <summary>
         /// Configuration key for locations to write internal diagnostic logs.
-        /// Comma-separated list, containing one of <c>file</c> or <c>datadog</c> e.g. file,datadog
```

Comment:
Why did you remove it, it was helpful? I'd leave the comment by saying that file is the only value accepted for now

Reply:
> Why did you remove it, it was helpful

Because there's no options currently 😂 But yeah, will do